Docs/source/developers/fields.rst

```diff
@@ -0,0 +1,121 @@
+Fields
+======
+
+.. note::
+
+   Add info on staggering and domain decomposition. Synchronize with section ``initialization``.
+
+The main fields are the electric field ``Efield``, the magnetic field ``Bfield``, the current density ``current`` and the charge density ``rho``. When a divergence-cleaner is used, another field ``F`` (containing ``div(Efield)-rho``).
+
+Due the AMR strategy used in WarpX (see section :doc:`../../theory/amr` for a complete description), each field on a given refinement level ``lev`` (except for the coarsest `0`) is defined on:
```

Comment:
You can add a section reference as indicated in the other files, e.g. call it `theory-amr`, and then link to it via:
```suggestion
Due the AMR strategy used in WarpX (see section :ref:`Theory: AMR <theory-amr>` for a complete description), each field on a given refinement level ``lev`` (except for the coarsest ``0``) is defined on:
```